liblumen_codegen/lib/lumen/compiler/Dialect/EIR/Conversion/EIRToLLVM/ConvertEIRToLLVM.cpp

```diff
@@ -932,20 +982,22 @@ struct ConstantBinaryOpConversion : public EIROpConversion<ConstantBinaryOp> {
     Value valPtrLoad = llvm_load(valPtr);
     Value header = llvm_constant(termTy, getIntegerAttr(rewriter, headerRaw));
     Value flags = llvm_constant(termTy, getIntegerAttr(rewriter, flagsRaw));
-    Value allocN = llvm_constant(termTy, rewriter.getI64IntegerAttr(1));
-    Value descAlloc = llvm_alloca(ptrTy, allocN, rewriter.getI64IntegerAttr(8));
+
+    int64_t size = (targetInfo.pointerSizeInBits / 8) * 3;
```

Comment:
i get the `/ 8` to get bits to bytes, but why `* 3`?

Reply:
The size of the struct here is 3 words. I need to write some reusable helpers to calculate struct sizes generically, just haven't done that yet for these types.